packages/next/src/client/lib/console.ts

```diff
@@ -0,0 +1,111 @@
+function formatObject(arg: unknown, depth: number) {
```

Comment:
Is this copied from somewhere?

Reply:
it started here: https://github.com/vercel/next.js/pull/71405